deploy/templates/nstemplatetiers/base1ns/cluster.yaml

```diff
@@ -116,6 +116,20 @@ objects:
       labels:
         matchLabels:
           toolchain.dev.openshift.com/space: ${SPACE_NAME}
+- apiVersion: rbac.authorization.k8s.io/v1
+  kind: ClusterRoleBinding
+  metadata:
+    name: osl-access-${SPACE_NAME}
+    annotations:
+      toolchain.dev.openshift.com/feature: openshift-lightspeed
+  roleRef:
+    apiGroup: rbac.authorization.k8s.io
+    kind: ClusterRole
+    name: lightspeed-operator-query-access
```

Comment:
if the ClusterRole doesn't exist it will not error or complain right ?

Reply:
Correct.